### ISSUE TITLE
Lower accuracy of PSPNet than the reported numbers of this repos

### ISSUE THREAD

**Issue by xiaomengyc** [hszhao/semseg#61]
Hi, I trained PSPNet with ResNet101 as the backbone. The mIoU values are 78.29/85.74/95.18 which are lower than the numbers in the readme file (0.7907/0.8636/0.9534). I use the same config as the author provided and do not change anything.  Does somebody get the same accuracy as the reported? My PyTorch version is 1.3.1. Don't know if PyTorch will affect the accuracy a little.

**Comment by swjtulinxi** [comment]
hi，i got the same problem,have you solved it

**Comment by xiaomengyc** [comment]
No, I haven't. Still do not where the problem is.

**Comment by swjtulinxi** [comment]
the author use resnet101  and  ms predict    ,,do you use the same  batchsize and  cropsize?

**Comment by xiaomengyc** [comment]
Yes, I use the default configuration. Batchsize and cropsize should be the same as the author's implementation.  Are your results same to my numbers?

**Comment by swjtulinxi** [comment]
my results is lower than yours,because i don't have enough gpu, i also know that many other people can 't reach author's result ,yours seems to be the best one  so far, i think your results seems that you don't  use  ms  predict,

**Comment by xiaomengyc** [comment]
Yes, I didn't use MS prediction, because the author's results 0.7907/0.8636/0.9534,  also do not use MS prediction. They are indicated by `ss` and `ms` in the tables.

**Comment by CharlesPikachu** [comment]
maybe you can try this repo: https://github.com/SegmentationBLWX/sssegmentation

**Comment by bea-CC** [comment]
Trained with PASCAL VOC2012 segmentation dataset, I got 0.7843/0.8593/0.9520（ss）、0.7938/0.8657/0.9540（ms）with TITAN Xp（4 GPUs）.
but I spent 10.5h， is it reasonable?
And I don't understand why the paper said "we achieve 82.6% accuracy.",  It's not the same as here.

**Comment by shashankskagnihotri** [comment]
I got Eval result: mIoU/mAcc/allAcc 0.6570/0.7378/0.9222 (ss) for PASCAL VOC2021 on 8 nvidia rtx 2080 gpus
Seems much lower. I used the default hyperparameters mentioned in the configs for pspnet50.
Did anyone try other hyperparameters and got better results?
